README.md

```diff
@@ -54,42 +82,43 @@ import magnolify.guava.semiauto._
 FunnelDerivation[MyCaseClass]
 ```
 
-Case class type conversion must be done explicitly.
+Case class type conversion must be called explicitly, specifying the case class used for conversion, rather than deriving entirely with implicits.
 
 ```scala
+// an annotated example with Avro GenericRecord
 import magnolify.avro._
-val t = AvroType[MyCaseClass]
-val r = t.to(MyCaseClass(...))
-val c = t.from(t)
+import org.apache.avro.generic.GenericRecord
+val typeclass = AvroType[MyCaseClass] // AvroType typeclass instance defines to and from conversions between Case Class and GenericRecord
+val genericRecord: GenericRecord = typeclass.to(MyCaseClass(...)) // instantiate a case class, then pass in
```

Comment:
Like the expanded docs here, helped me understand this part more. 

I think it wouldn't hurt to make it more explicit, e.g. define 
```scala
case class MyCaseClass(foo: String, bar: Int)=
```
and then pass in actual args here 
```scals
val genericRecord: GenericRecord = typeclass.to(MyCaseClass("hello", 1)) // instantiate a case class, then pass in
```